css/store.css

```diff
@@ -0,0 +1,194 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+.header-container-store {
+    background-color: rgb(245, 244, 244);
+    border-bottom: 1px solid rgb(235, 235, 235);
+}
+
+main {
+    display: flex;
+    border-top: solid 0.03rem lightgrey;
+}
+
+.required:after {
+    content: " *";
+}
+
+/* 2 sections in the main get 50% width each */
+main > section {
+    width: 50%;
+}
+
+main h1 {
+    color: #DE6E49;
+    padding: 1.5rem 8rem ;
+    size: 15rem;
+    font-weight: 500;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    color:rgb(87, 81, 81);
+    height:113vh;
+}
+
+form input,
+form select {
+    border: 1px solid gray;
+    border-radius: 4px;
+    height: 2rem;
+    margin-bottom: 1.3rem;
+}
+
+label {
+    padding-bottom: 0.5rem;
+}
+
+.flex-field {
+    display :flex;
+    justify-content: center;   
+}
+
+.first-name-container {
+    display: flex;
+    flex-direction: column;
+    padding: 0.5rem 0.9rem;
+    width: 10.8rem;
+}
+
+.address-container {
+    display: flex;
+    flex-direction: column;
+    padding: 0 7.7rem;
```

Comment:
Having padding 7.7rem set here makes the alignment not quite right  - when you come to use css-grid it will be easier as the grid items will fill the space 🙂